doc/codingGuidelines.md

```diff
@@ -221,14 +221,15 @@ Respect the minimum supported version for Maya and USD stated in [build.md](http
 Recent extensions to the C++ standard introduce many features previously only found in [boost](http://boost.org). To avoid introducing additional dependencies, developers should strive to use functionality in the C++ std over boost. If you encounter usage of boost in the code, consider converting this to the equivalent std mechanism. 
 Our library currently has the following boost dependencies:
 * `boost::python`
-* `boost::hash_combine` (see  [this proposal](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2017/p0814r0.pdf) )
 * `boost::make_shared` (preferable to replace with `std::shared_ptr`)
 
 ***Update:***
```

Comment:
In the last PR, it would be good to clean-up this Update section, since we people can use GitHub to check the history of changes. We don't need to keep them in the document.